src/components/Input.tsx

```diff
@@ -26,6 +26,7 @@ export enum InputType {
 
 type StyleProps = {
   className?: string;
+  $backgroundColorOverride?: string;
```

Comment:
i think we can do that it might be a bit more involved though. there's a lot of styling around the background color for the Input. we even have an !important tag 😭.
``` 
&[data-autocompleted] {
    background-color: var(--input-backgroundColor) !important;
  }
```

you mind if i make a ticket to investigate simplifying the input styles? cause it mightttt not be a super quick fix

Reply:
Yeah totally fine with keeping it for now